ext/imap/tests/imap_constructor.phpt

```diff
@@ -0,0 +1,15 @@
+--TEST--
+Attempt to instantiate an IMAPConnection directly
+--SKIPIF--
+<?php
+require_once __DIR__ . '/setup/skipif.inc';
```

Comment:
Just a nitpick but this test doesn't need to check for the presence of a mailbox, so could you change this to
```suggestion
<?php
extension_loaded('imap') or die('skip imap extension not available in this build');
```

Reply:
Thank you, I just force-pushed an updated commit with this test simplified like you suggested.